packages/flutter_tools/lib/src/commands/ios.dart

```diff
@@ -0,0 +1,164 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+import '../runner/flutter_command_runner.dart';
+import '../runner/flutter_command.dart';
+
+class IOSCommand extends FlutterCommand {
+  final String name = "ios";
```

Comment:
Is this specific to iOS development with Xcode? Or is it inconceivable to develop for iOS without Xcode?